ui/pages/confirmations/components/confirm/info/hooks/useEIP1559TxFees.ts

```diff
@@ -9,11 +10,14 @@ export const useEIP1559TxFees = (
   maxPriorityFeePerGas: string;
 } => {
   const hexMaxFeePerGas =
-    transactionMeta.dappSuggestedGasFees?.maxFeePerGas ||
-    transactionMeta?.txParams?.maxFeePerGas;
+    transactionMeta.userFeeLevel === PriorityLevels.dAppSuggested
```

Comment:
If the intent of the confirmation is ultimately to render the underlying transaction params that will be submitted, then why are we not always rendering the values from `txParams`?

Should the gas fee modal not be updating the underlying params when a new priority level is selected?

At what point are they put in the params that get submitted?

Reply:
That's a good point, I got confused. We don't need to display the dApp suggested fees. Using the values from txParams should suffice because they are updated with the user selection on the gas modals.